tests/gtest/avifdecodetest.cc

```diff
@@ -41,6 +41,25 @@ TEST(AvifDecodeTest, ParseEmptyData) {
   ASSERT_EQ(avifDecoderParse(decoder.get()), AVIF_RESULT_INVALID_FTYP);
 }
 
+// From https://crbug.com/334281983.
+TEST(AvifDecodeTest, PeekCompatibleFileTypeBad1) {
+  constexpr uint8_t kData[] = {0x00, 0x00, 0x00, 0x1c, 0x66,
+                               0x74, 0x79, 0x70, 0x84, 0xca};
+  constexpr size_t kSize = 10;
```

Comment:
I'm not suggesting a change, but note std::array or sizeof could work here.

Reply:
Done. Replaced `kSize` with `sizeof(kData)`. Thanks for the suggestion.